example/android/app/src/profile/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.mapbox.example">
```

Comment:
Let's add the package again `package="com.mapbox.example`.

Reply:
Added it again ✅